<?xml version="1.0" encoding="utf-8"?>
<!DOCTYPE TS>
<TS version="2.1" language="it">
<context>
    <name>API</name>
    <message>
        <location filename="../qgiscloudapi/qgiscloudapi.py" line="105"/>
        <source>Failed to access QGIS Cloud API on {api_url}</source>
        <translation type="unfinished"></translation>
    </message>
</context>
<context>
    <name>BackgroundLayersMenu</name>
    <message>
        <location filename="../background_layers_menu.py" line="219"/>
        <source>Error</source>
        <translation>Errore</translation>
    </message>
    <message>
        <location filename="../background_layers_menu.py" line="219"/>
        <source>A serious error has occurred during the coordinate transformation. Please set the reference system of the project to the WGS84 / Pseudo-Mercartor Projektion (EPSG: 3857) and reload the layer.</source>
        <translation>Si è verificato un grave errore durante la trasformazione delle coordinate. Per favore, imposta il sistema di riferimento del progetto sul WGS84 / Pseudo-Mercartor Projektion (EPSG: 3857) e ricarica il livello.</translation>
    </message>
</context>
<context>
    <name>DataUpload</name>
    <message>
        <location filename="../data_upload.py" line="69"/>
        <source>Uploading to database &apos;{db}&apos;...</source>
        <translation>Caricando nel database &apos;{db}&apos;...</translation>
    </message>
    <message>
        <location filename="../data_upload.py" line="86"/>
        <source>Database full</source>
        <translation>Database pieno</translation>
    </message>
    <message>
        <location filename="../data_upload.py" line="86"/>
        <source>You have exceeded the maximum database size for your current QGIS Cloud plan. Please free up some space or upgrade your QGIS Cloud plan.</source>
        <translation>Ê stato ecceduto la dimensione massima del database per il piano QGIS Cloud attualmente in uso. Per favore liberare dello spazio oppure scegliere un piano con più capacità.</translation>
    </message>
    <message>
        <location filename="../data_upload.py" line="129"/>
        <source>Creating table &apos;{table}&apos;...</source>
        <translation>Creando tabella &apos;{table}&apos;...</translation>
    </message>
    <message>
        <location filename="../data_upload.py" line="179"/>
        <source>Uploading features...</source>
        <translation>Caricando oggetti...</translation>
    </message>
    <message>
        <location filename="../data_upload.py" line="193"/>
        <source>Feature {id} of layer {layer} has no geometry</source>
        <translation>Oggetto {id} del layer {layer} non ha geometrie</translation>
    </message>
    <message>
        <location filename="../data_upload.py" line="244"/>
        <source>{table}: {count} features uploaded</source>
        <translation>{table}: {count} oggetti caricati</translation>
    </message>
    <message>
        <location filename="../data_upload.py" line="196"/>
        <source>Feature {id} of layer {layer} has wrong geometry type {type}</source>
        <translation>L&apos;oggetto {id} del Layer {layer} ha un tipo di geometria incorretto {type}</translation>
    </message>
</context>
<context>
    <name>DbConnectionCfg</name>
    <message>
        <location filename="../db_connection_cfg.py" line="184"/>
        <source>DB Connection Failed</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../db_connection_cfg.py" line="184"/>
        <source>Could not connect to the Database &lt;b&gt;%s&lt;/b&gt;&lt;br&gt;&lt;br&gt;. 

Please check the following points:&lt;br&gt;&lt;br&gt;

- Is the computer connected to the Internet?&lt;br&gt;
- Is port 5432 open in the firewall?&lt;br&gt;
- Are the proxy settings correct?</source>
        <translation type="unfinished"></translation>
    </message>
</context>
<context>
    <name>ErrorReportDialog</name>
    <message>
        <location filename="../error_report_dialog.py" line="54"/>
        <source>Report error</source>
        <translation>Segnala errore</translation>
    </message>
</context>
<context>
    <name>LoginDialog</name>
    <message>
        <location filename="../ui_login.ui" line="14"/>
        <source>Login</source>
        <translation>Accedere</translation>
    </message>
    <message>
        <location filename="../ui_login.ui" line="20"/>
        <source>User:</source>
        <translation>Nome utente:</translation>
    </message>
    <message>
        <location filename="../ui_login.ui" line="43"/>
        <source>Password:</source>
        <translation>Password:</translation>
    </message>
    <message>
        <location filename="../ui_login.ui" line="72"/>
        <source>Token:</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../ui_login.ui" line="86"/>
        <source>Copy your token from &lt;a href=&quot;{token_page_url}&quot;&gt;here&lt;/a&gt;</source>
        <translation type="unfinished"></translation>
    </message>
</context>
<context>
    <name>MapSettingsDialog</name>
    <message>
        <location filename="../mapsettingsdialog.py" line="45"/>
        <source>Map settings for map: %s</source>
        <translation>Impostazioni della mappa per la mappa: %s</translation>
    </message>
    <message>
        <location filename="../mapsettingsdialog.py" line="228"/>
        <source>Sql Error</source>
        <translation>Errore Sql</translation>
    </message>
    <message>
        <location filename="../mapsettingsdialog.py" line="280"/>
        <source>Add user(s)</source>
        <translation>Aggiungi utente(i)</translation>
    </message>
    <message>
        <location filename="../mapsettingsdialog.py" line="280"/>
        <source>Usernames:</source>
        <translation>Nomi utente</translation>
    </message>
    <message>
        <location filename="../mapsettingsdialog.py" line="313"/>
        <source>User error</source>
        <translation>Errore dell&apos;utente</translation>
    </message>
    <message>
        <location filename="../mapsettingsdialog.py" line="313"/>
        <source>The user(s): %s don&apos;t exist.</source>
        <translation>L&apos;utente o gli utenti: %s non esistono.</translation>
    </message>
    <message>
        <location filename="../mapsettingsdialog.py" line="377"/>
        <source>Success</source>
        <translation>Successo</translation>
    </message>
    <message>
        <location filename="../mapsettingsdialog.py" line="321"/>
        <source>The user(s): %s have been added.</source>
        <translation>L&apos;utente(i): %s è stato aggiunto.</translation>
    </message>
    <message>
        <location filename="../mapsettingsdialog.py" line="329"/>
        <source>Error</source>
        <translation>Errore</translation>
    </message>
    <message>
        <location filename="../mapsettingsdialog.py" line="329"/>
        <source>No username given.</source>
        <translation>Nessun nome utente indicato.</translation>
    </message>
    <message>
        <location filename="../mapsettingsdialog.py" line="359"/>
        <source>Do you really want to delete the user: %s?</source>
        <translation>Vuoi davvero cancellare l&apos;utente: %s?</translation>
    </message>
    <message>
        <location filename="../mapsettingsdialog.py" line="377"/>
        <source>The user: %s has been deleted.</source>
        <translation>L&apos;utente: %s è stato cancellato.</translation>
    </message>
</context>
<context>
    <name>QgisCloudPlugin</name>
    <message>
        <location filename="../ui_qgiscloudplugin.py" line="380"/>
        <source>Upload Data</source>
        <translation>Caricamento dati</translation>
    </message>
    <message>
        <location filename="../ui_qgiscloudplugin.py" line="345"/>
        <source>Add background layer</source>
        <translation>Aggiungere layer di sfondo</translation>
    </message>
    <message>
        <location filename="../ui_qgiscloudplugin.py" line="346"/>
        <source>&lt;i&gt;To add a background layer to the map, install the OpenLayers plugin.&lt;/i&gt;</source>
        <translation>&lt;i&gt;Per aggiungere un layer di sfondo, installare il plugin OpenLayers.&lt;/i&gt;</translation>
    </message>
    <message>
        <location filename="../ui_qgiscloudplugin.py" line="347"/>
        <source>Publish Map</source>
        <translation>Pubblicare cartina</translation>
    </message>
    <message>
        <location filename="../ui_qgiscloudplugin.py" line="361"/>
        <source>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:&apos;Sans Serif&apos;; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;a href=&quot;mailto:support@qgiscloud.com?subject=QGISCloud support&quot;&gt;&lt;span style=&quot; font-family:&apos;Ubuntu&apos;; text-decoration: underline; color:#0057ae;&quot;&gt;support@qgiscloud.com&lt;/span&gt;&lt;/a&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</source>
        <translation>&amp;lt;!DOCTYPE HTML PUBLIC &amp;quot;-//W3C//DTD HTML 4.0//EN&amp;quot; &amp;quot;http://www.w3.org/TR/REC-html40/strict.dtd&amp;quot;&amp;gt;
&amp;lt;html&amp;gt;&amp;lt;head&amp;gt;&amp;lt;meta name=&amp;quot;qrichtext&amp;quot; content=&amp;quot;1&amp;quot; /&amp;gt;&amp;lt;style type=&amp;quot;text/css&amp;quot;&amp;gt;
p, li { white-space: pre-wrap; }
&amp;lt;/style&amp;gt;&amp;lt;/head&amp;gt;&amp;lt;body style=&amp;quot; font-family:&apos;Sans Serif&apos;; font-size:9pt; font-weight:400; font-style:normal;&amp;quot;&amp;gt;
&amp;lt;p style=&amp;quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&amp;quot;&amp;gt;&amp;lt;a href=&amp;quot;mailto:support@qgiscloud.com?subject=QGISCloud support&amp;quot;&amp;gt;&amp;lt;span style=&amp;quot; font-family:&apos;Ubuntu&apos;; text-decoration: underline; color:#0057ae;&amp;quot;&amp;gt;support@qgiscloud.com&amp;lt;/span&amp;gt;&amp;lt;/a&amp;gt;&amp;lt;/p&amp;gt;&amp;lt;/body&amp;gt;&amp;lt;/html&amp;gt;</translation>
    </message>
    <message>
        <location filename="../ui_qgiscloudplugin.py" line="377"/>
        <source>Database:</source>
        <translation>Database:</translation>
    </message>
    <message>
        <location filename="../ui_qgiscloudplugin.py" line="378"/>
        <source>Refresh layers</source>
        <translation>Aggiornare layer</translation>
    </message>
    <message>
        <location filename="../ui_qgiscloudplugin.py" line="379"/>
        <source>Upload data</source>
        <translation>Caricare dati</translation>
    </message>
    <message>
        <location filename="../ui_qgiscloudplugin.py" line="382"/>
        <source>https://api.qgiscloud.com</source>
        <translation>https://api.qgiscloud.com</translation>
    </message>
    <message>
        <location filename="../ui_qgiscloudplugin.py" line="383"/>
        <source>Reset QGIS Cloud API URL</source>
        <translation>Resettare URL QGIS Cloud API</translation>
    </message>
    <message>
        <location filename="../ui_qgiscloudplugin.py" line="384"/>
        <source>...</source>
        <translation>...</translation>
    </message>
    <message>
        <location filename="../ui_qgiscloudplugin.py" line="385"/>
        <source>Login</source>
        <translation>Accedere</translation>
    </message>
    <message>
        <location filename="../ui_qgiscloudplugin.py" line="386"/>
        <source>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:&apos;Ubuntu&apos;; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;a href=&quot;https://qgiscloud.com/account/sign_up&quot;&gt;&lt;span style=&quot; text-decoration: underline; color:#0057ae;&quot;&gt;Signup&lt;/span&gt;&lt;/a&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</source>
        <translation>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:&apos;Ubuntu&apos;; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;a href=&quot;https://qgiscloud.com/account/sign_up&quot;&gt;&lt;span style=&quot; text-decoration: underline; color:#0057ae;&quot;&gt;Registrarsi&lt;/span&gt;&lt;/a&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</translation>
    </message>
    <message>
        <location filename="../ui_qgiscloudplugin.py" line="391"/>
        <source>Logged in as ...</source>
        <translation>Collegato come ...</translation>
    </message>
    <message>
        <location filename="../ui_qgiscloudplugin.py" line="392"/>
        <source>Logout</source>
        <translation>Logout</translation>
    </message>
    <message>
        <location filename="../ui_qgiscloudplugin.py" line="393"/>
        <source>Databases</source>
        <translation>Database</translation>
    </message>
    <message>
        <location filename="../ui_qgiscloudplugin.py" line="394"/>
        <source>Create</source>
        <translation>Crea</translation>
    </message>
    <message>
        <location filename="../ui_qgiscloudplugin.py" line="395"/>
        <source>Delete</source>
        <translation>Elimina</translation>
    </message>
    <message>
        <location filename="../ui_qgiscloudplugin.py" line="397"/>
        <source>Refresh</source>
        <translation>Aggiorna</translation>
    </message>
    <message>
        <location filename="../ui_qgiscloudplugin.py" line="398"/>
        <source>Account</source>
        <translation>Conto utente</translation>
    </message>
    <message>
        <location filename="../ui_qgiscloudplugin.py" line="399"/>
        <source>&lt;b&gt;Plugin version:&lt;/b&gt;</source>
        <translation>&lt;b&gt;Versione plugin:&lt;/b&gt;</translation>
    </message>
    <message>
        <location filename="../ui_qgiscloudplugin.py" line="400"/>
        <source>About</source>
        <translation>Informazioni</translation>
    </message>
    <message>
        <location filename="../ui_qgiscloudplugin.py" line="376"/>
        <source>Maps</source>
        <translation>Mappe</translation>
    </message>
    <message>
        <location filename="../ui_qgiscloudplugin.py" line="349"/>
        <source>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:&apos;Ubuntu&apos;; font-size:11pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;a href=&quot;http://wms.qgiscloud.com/user/map&quot;&gt;&lt;span style=&quot; font-size:9pt; text-decoration: underline; color:#0057ae;&quot;&gt;https://wms.qgiscloud.com/user/map&lt;/span&gt;&lt;/a&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</source>
        <translation></translation>
    </message>
    <message>
        <location filename="../ui_qgiscloudplugin.py" line="355"/>
        <source>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:&apos;Ubuntu&apos;; font-size:11pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;a href=&quot;http://qgiscloud.com/maps&quot;&gt;&lt;span style=&quot; font-size:9pt; text-decoration: underline; color:#0057ae;&quot;&gt;https://qgiscloud.com/maps&lt;/span&gt;&lt;/a&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</source>
        <translation></translation>
    </message>
    <message>
        <location filename="../ui_qgiscloudplugin.py" line="367"/>
        <source>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:&apos;Ubuntu&apos;; font-size:11pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;a href=&quot;http://qgiscloud.com/user/map&quot;&gt;&lt;span style=&quot; font-size:9pt; text-decoration: underline; color:#0057ae;&quot;&gt;https://qgiscloud.com/user/map&lt;/span&gt;&lt;/a&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</source>
        <translation></translation>
    </message>
    <message>
        <location filename="../ui_qgiscloudplugin.py" line="373"/>
        <source>Published Maps</source>
        <translation>Mappe pubblicate</translation>
    </message>
    <message>
        <location filename="../ui_qgiscloudplugin.py" line="374"/>
        <source>Open Project</source>
        <translation>Apri Progetto</translation>
    </message>
    <message>
        <location filename="../ui_qgiscloudplugin.py" line="381"/>
        <source>&amp;Server:</source>
        <translation></translation>
    </message>
    <message>
        <location filename="../ui_qgiscloudplugin.py" line="344"/>
        <source>QGIS Cloud</source>
        <translation>QGIS Cloud</translation>
    </message>
    <message>
        <location filename="../ui_qgiscloudplugin.py" line="375"/>
        <source>Map Settings</source>
        <translation>Impostazioni della mappa</translation>
    </message>
    <message>
        <location filename="../ui_qgiscloudplugin.py" line="348"/>
        <source>Public WMS/WFS/WCS:</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../ui_qgiscloudplugin.py" line="354"/>
        <source>Map Admin:</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../ui_qgiscloudplugin.py" line="360"/>
        <source>Support:</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../ui_qgiscloudplugin.py" line="366"/>
        <source>Webmap:</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../ui_qgiscloudplugin.py" line="396"/>
        <source>DB Tables</source>
        <translation type="unfinished"></translation>
    </message>
</context>
<context>
    <name>QgisCloudPluginDialog</name>
    <message>
        <location filename="../qgiscloudplugindialog.py" line="1045"/>
        <source>QGIS Cloud</source>
        <translation>QGIS Cloud</translation>
    </message>
    <message>
        <location filename="../qgiscloudplugindialog.py" line="430"/>
        <source>Unsupported versions detected. Please check your versions first!</source>
        <translation>Versioni non supportate rilevate. Verificare le versioni in uso!</translation>
    </message>
    <message>
        <location filename="../qgiscloudplugindialog.py" line="460"/>
        <source>Login failed</source>
        <translation>Accesso fallito</translation>
    </message>
    <message>
        <location filename="../qgiscloudplugindialog.py" line="451"/>
        <source>Wrong user name or password</source>
        <translation>Nome utente o password incorretta</translation>
    </message>
    <message>
        <location filename="../qgiscloudplugindialog.py" line="562"/>
        <source>Delete QGIS Cloud database.</source>
        <translation>Eliminare database QGIS Cloud.</translation>
    </message>
    <message>
        <location filename="../qgiscloudplugindialog.py" line="658"/>
        <source>No databases</source>
        <translation>Nessun database</translation>
    </message>
    <message>
        <location filename="../qgiscloudplugindialog.py" line="668"/>
        <source>Select database</source>
        <translation>Selezionare database</translation>
    </message>
    <message>
        <location filename="../qgiscloudplugindialog.py" line="851"/>
        <source>The project has been modified.</source>
        <translation>Il progetto è stato modificato.</translation>
    </message>
    <message>
        <location filename="../qgiscloudplugindialog.py" line="1008"/>
        <source>Cancelled</source>
        <translation>Annullato</translation>
    </message>
    <message>
        <location filename="../qgiscloudplugindialog.py" line="1010"/>
        <source>Publishing map</source>
        <translation>Pubblicando cartina</translation>
    </message>
    <message>
        <location filename="../qgiscloudplugindialog.py" line="1047"/>
        <source>Map successfully published</source>
        <translation>Cartina pubblicata con successo</translation>
    </message>
    <message>
        <location filename="../qgiscloudplugindialog.py" line="1051"/>
        <source>Error uploading project</source>
        <translation>Errore durante il caricamento del progetto</translation>
    </message>
    <message>
        <location filename="../qgiscloudplugindialog.py" line="1057"/>
        <source>Uploading SVG symbols</source>
        <translation>Caricando simboli SVG</translation>
    </message>
    <message>
        <location filename="../qgiscloudplugindialog.py" line="1096"/>
        <source>Error checking local data sources</source>
        <translation>Errore durante la scansione delle fonti dati locali</translation>
    </message>
    <message>
        <location filename="../qgiscloudplugindialog.py" line="1108"/>
        <source>Local layers found</source>
        <translation>Layer locali trovati</translation>
    </message>
    <message>
        <location filename="../qgiscloudplugindialog.py" line="1113"/>
        <source>Unsupported layers found</source>
        <translation>Layer non supportati trovati</translation>
    </message>
    <message>
        <location filename="../qgiscloudplugindialog.py" line="1117"/>
        <source>
Please remove or replace above layers before publishing your map.
</source>
        <translation>
Per favore rimuovere o sostituire i layer sopra elencati prima di pubblicare la cartina.</translation>
    </message>
    <message>
        <location filename="../qgiscloudplugindialog.py" line="1175"/>
        <source>Unsupported geometry type</source>
        <translation>Tipo di geometria nono supporati</translation>
    </message>
    <message>
        <location filename="../qgiscloudplugindialog.py" line="1175"/>
        <source>Unsupported geometry type &apos;{type}&apos; in layer &apos;{layer}&apos;</source>
        <translation>Il tipo di geometria &apos;{type}&apos; nel layer &apos;{layer&apos;} non è supportato</translation>
    </message>
    <message>
        <location filename="../qgiscloudplugindialog.py" line="1181"/>
        <source>Note: OGR features will be converted to MULTI-type</source>
        <translation>Nota: Oggetti OGR verranno convertiti in geometrie MULTI-tipo</translation>
    </message>
    <message>
        <location filename="../qgiscloudplugindialog.py" line="1235"/>
        <source>Updated local data sources</source>
        <translation>Fonti dati locali attualizzate</translation>
    </message>
    <message>
        <location filename="../qgiscloudplugindialog.py" line="1271"/>
        <source>Unknown type</source>
        <translation>Tipo sconosciuto</translation>
    </message>
    <message>
        <location filename="../qgiscloudplugindialog.py" line="1391"/>
        <source>No database available</source>
        <translation>Nessun database disponibile</translation>
    </message>
    <message>
        <location filename="../qgiscloudplugindialog.py" line="1391"/>
        <source>Please create a database in the &apos;Account&apos; tab.</source>
        <translation>Per favore creare un database nella scheda &apos;Conto utente&apos;.</translation>
    </message>
    <message>
        <location filename="../qgiscloudplugindialog.py" line="1396"/>
        <source>No database selected</source>
        <translation>Nessun database selezionato</translation>
    </message>
    <message>
        <location filename="../qgiscloudplugindialog.py" line="1396"/>
        <source>Please select a database to upload data.</source>
        <translation>Per fabore selezionare un database per caricare dati.</translation>
    </message>
    <message>
        <location filename="../qgiscloudplugindialog.py" line="1406"/>
        <source>Network Error</source>
        <translation>Errore di rete</translation>
    </message>
    <message>
        <location filename="../qgiscloudplugindialog.py" line="1413"/>
        <source>Uploading data...</source>
        <translation>Caricando dati...</translation>
    </message>
    <message>
        <location filename="../qgiscloudplugindialog.py" line="1505"/>
        <source>QGIS Cloud Error</source>
        <translation>Errore QGIS Cloud</translation>
    </message>
    <message>
        <location filename="../qgiscloudplugindialog.py" line="1507"/>
        <source>Error</source>
        <translation>Errore</translation>
    </message>
    <message>
        <location filename="../qgiscloudplugindialog.py" line="1592"/>
        <source>Used DB Storage: </source>
        <translation>Spazio DB usato: </translation>
    </message>
    <message>
        <location filename="../db_connection_cfg.py" line="138"/>
        <source>host: %s port: %s database: %s username: %s password: %s</source>
        <translation>host: %s port: %s database: %s username: %s password: %s</translation>
    </message>
    <message>
        <location filename="../qgiscloudplugindialog.py" line="850"/>
        <source>Project Modified</source>
        <translation>Progetto modificato</translation>
    </message>
    <message>
        <location filename="../qgiscloudplugindialog.py" line="1096"/>
        <source>An error occured.</source>
        <translation>SI è verificato un errore</translation>
    </message>
    <message>
        <location filename="../qgiscloudplugindialog.py" line="1441"/>
        <source>Upload errors occurred</source>
        <translation>Errori durante il caricamento dei dati</translation>
    </message>
    <message>
        <location filename="../qgiscloudplugindialog.py" line="1441"/>
        <source>Upload errors occurred. Not all data could be uploaded.</source>
        <translation>Si sono verificati degli errori durante il caricamento dei dati. Non tutti i dati sono stati caricati.</translation>
    </message>
    <message>
        <location filename="../qgiscloudplugindialog.py" line="1478"/>
        <source>Save Project</source>
        <translation>Salvare projetto</translation>
    </message>
    <message>
        <location filename="../qgiscloudplugindialog.py" line="1469"/>
        <source>Choose were to save the modified project:</source>
        <translation>Scegliere dove salvare il progetto modificato:</translation>
    </message>
    <message>
        <location filename="../qgiscloudplugindialog.py" line="1478"/>
        <source>QGIS Project Files</source>
        <translation>Progetti QGIS</translation>
    </message>
    <message>
        <location filename="../qgiscloudplugindialog.py" line="1465"/>
        <source>Upload complete. The local layers in the project were replaced with the layers uploaded to the qgiscloud database.</source>
        <translation>Caricamento dei dati completato. I layer locali nel progetto sono stati sostituiti con i layer caricati nel database qgiscloud.</translation>
    </message>
    <message>
        <location filename="../qgiscloudplugindialog.py" line="852"/>
        <source>The project needs to be saved before it can be published. Proceed?</source>
        <translation>Il progetto deve essere salvato prima che può essere pubblicato. Procedere?</translation>
    </message>
    <message>
        <location filename="../qgiscloudplugindialog.py" line="455"/>
        <source>Login failed: %s</source>
        <translation>Accesso fallito: %s</translation>
    </message>
    <message>
        <location filename="../qgiscloudplugindialog.py" line="983"/>
        <source>Warning!</source>
        <translation>Attenzione!</translation>
    </message>
    <message>
        <location filename="../qgiscloudplugindialog.py" line="514"/>
        <source>Number of %s permitted databases exceeded! Please upgrade your account!</source>
        <translation>Il numero dei Database %s permessi è stato superato. Per favore acquistate ulteriori database!</translation>
    </message>
    <message>
        <location filename="../qgiscloudplugindialog.py" line="938"/>
        <source>The project has a user defined CRS. The use of user defined CRS is not supported. Please correct the project CRS before publishing!</source>
        <translation>Nel progetto è impostato un sistema di riferimento personalizzato. QGIS Cloud non supporta systemi di riferimento personalizzati. Per favore correggere il sistema di riferimento prima di pubblicare!</translation>
    </message>
    <message>
        <location filename="../qgiscloudplugindialog.py" line="1406"/>
        <source>Could not connect to database server ({0}) on port {1}. Please contact your system administrator or internet provider to open port {1} in the firewall</source>
        <translation>Impossibile connettersi al server database ({0}) usando la porta {1}. Contattare l&apos;amministratore del sistema o il provider internet per aprire la porta {1} nella firewall</translation>
    </message>
    <message>
        <location filename="../qgiscloudplugindialog.py" line="401"/>
        <source>New Version</source>
        <translation>Nuova versione</translation>
    </message>
    <message>
        <location filename="../qgiscloudplugindialog.py" line="401"/>
        <source>New plugin release {version} is available! Please upgrade the QGIS Cloud plugin.</source>
        <translation>Una nuova versione {version} del plugin è disponibile. Per favore aggiornare il plugin QGIS Cloud.</translation>
    </message>
    <message>
        <location filename="../qgiscloudplugindialog.py" line="438"/>
        <source>Account Disabled</source>
        <translation>Account disattivato</translation>
    </message>
    <message>
        <location filename="../qgiscloudplugindialog.py" line="438"/>
        <source>Account {username} is disabled! Please contact support@qgiscloud.com</source>
        <translation>L&apos;account {username} è disattivato. Per favore contattare support@qgiscloud.com</translation>
    </message>
    <message>
        <location filename="../qgiscloudplugindialog.py" line="447"/>
        <source>Login for user {username} failed</source>
        <translation>Login per l&apos;utente {username} fallito</translation>
    </message>
    <message>
        <location filename="../qgiscloudplugindialog.py" line="1002"/>
        <source>Warning</source>
        <translation>Attenzione</translation>
    </message>
    <message>
        <location filename="../qgiscloudplugindialog.py" line="1109"/>
        <source>Some layers are using local data. Please upload local layers to your cloud database in the &apos;Upload Data&apos; tab before publishing.

</source>
        <translation>Alcuni Layer utilizzano dati locali. Per favore caricare questi dati nel Database tramite la scheda &apos;Caricamento dati&apos; prima di pubblicare il progetto.</translation>
    </message>
    <message>
        <location filename="../qgiscloudplugindialog.py" line="160"/>
        <source>Privacy Policy</source>
        <translation>Informativa sulla privacy</translation>
    </message>
    <message>
        <location filename="../qgiscloudplugindialog.py" line="378"/>
        <source>Accept new Privacy Policy</source>
        <translation>Accetta la nuova Informativa sulla privacy</translation>
    </message>
    <message>
        <location filename="../qgiscloudplugindialog.py" line="411"/>
        <source>Logged in as {0} ({1})</source>
        <translation>Effettuato il login come {0} ({1})</translation>
    </message>
    <message>
        <location filename="../qgiscloudplugindialog.py" line="422"/>
        <source>Logged in as {0}</source>
        <translation>Effettuato il login come {0}</translation>
    </message>
    <message>
        <location filename="../qgiscloudplugindialog.py" line="544"/>
        <source>You have layers from database &quot;{name}&quot; loaded in your project! Do you want to remove them before you delete database &quot;{name}&quot;?</source>
        <translation>Nel progetto sono caricati i livelli del database &amp;quot;{name}&amp;quot;! Vuoi rimuoverli prima di eliminare il database &amp;quot;{name}&amp;quot;?</translation>
    </message>
    <message>
        <location filename="../qgiscloudplugindialog.py" line="557"/>
        <source>Deletion of database &quot;{name}&quot; interrupted!</source>
        <translation>Cancellazione della banca dati &amp;quot;{name}&amp;quot; interrotta!</translation>
    </message>
    <message>
        <location filename="../qgiscloudplugindialog.py" line="563"/>
        <source>Do you want to delete the database &quot;%s&quot;?</source>
        <translation>Vuoi cancellare il database &amp;quot;%s&amp;quot;?</translation>
    </message>
    <message>
        <location filename="../qgiscloudplugindialog.py" line="689"/>
        <source>Your actual project has changes. Do you want to save the project?</source>
        <translation>Il vostro progetto attuale ha delle modifiche. Vuoi salvare il progetto?</translation>
    </message>
    <message>
        <location filename="../qgiscloudplugindialog.py" line="712"/>
        <source>Delete QGIS Cloud map.</source>
        <translation>Elimina mappa QGIS Cloud.</translation>
    </message>
    <message>
        <location filename="../qgiscloudplugindialog.py" line="713"/>
        <source>Do you want to delete the map &quot;%s&quot;?</source>
        <translation>Vuoi cancellare la mappa &amp;quot;%s&amp;quot;?</translation>
    </message>
    <message>
        <location filename="../qgiscloudplugindialog.py" line="736"/>
        <source>Deletion of map &quot;{name}&quot; interrupted!</source>
        <translation>Cancellazione della mappa &amp;quot;{name}&amp;quot; interrotta!</translation>
    </message>
    <message>
        <location filename="../qgiscloudplugindialog.py" line="1115"/>
        <source>  -  %s (%s)
</source>
        <translation></translation>
    </message>
    <message>
        <location filename="../qgiscloudplugindialog.py" line="378"/>
        <source>Due to the GDPR qgiscloud.com has a new &lt;a href=&apos;http://qgiscloud.com/en/pages/privacy&apos;&gt; Privacy Policy &lt;/a&gt;.
                                To continue using qgiscloud.com, you must accept the new policy. </source>
        <translation>A causa del GDPR qgiscloud.com ha una nuova &amp;lt;a href=&apos;http://qgiscloud.com/en/pages/privacy&apos;&amp;gt; Informativa sulla privacy &amp;lt;/a&amp;gt;.
                                Per continuare ad utilizzare qgiscloud.com, è necessario accettare la nuova politica. </translation>
    </message>
    <message>
        <location filename="../qgiscloudplugindialog.py" line="934"/>
        <source>The project has an unknown CRS. Please set a valid map CRS in Project-&gt;Properties...-&gt;CRS before publishing!</source>
        <translation>Il progetto ha un CRS sconosciuto. Imposta un CRS valido per la mappa in Progetto-&amp;gt;Proprietà...-&amp;gt;CRS prima di pubblicare!</translation>
    </message>
    <message>
        <location filename="../qgiscloudplugindialog.py" line="973"/>
        <source>The following WMS layer names are not unique: {}. Please make sure the names are unique, then publish the project again. The layer WMS short name can be set in the layer properties dialog under &apos;QGIS Server -&gt; Short name&apos;.</source>
        <translation>I seguenti nomi di livelli WMS non sono unici: {}. Assicurati che i nomi siano unici, poi pubblica di nuovo il progetto. Il nome breve del livello WMS può essere impostato nella finestra di dialogo delle proprietà del livello sotto &apos;QGIS Server -&amp;gt; Nome breve&apos;.</translation>
    </message>
    <message>
        <location filename="../qgiscloudplugindialog.py" line="467"/>
        <source>Could not resolve Host Name</source>
        <translation>Impossibile risolvere il nome dell&apos;host</translation>
    </message>
    <message>
        <location filename="../qgiscloudplugindialog.py" line="467"/>
        <source>Could not access {url}. Please check that the URL is written correctly. The error: was &apos;{e}&apos;</source>
        <translation>Impossibile accedere a {url}. Per favore controlla che l&apos;URL sia scritto correttamente. L&apos;errore: era &apos;{e}&apos;</translation>
    </message>
    <message>
        <location filename="../qgiscloudplugindialog.py" line="477"/>
        <source>No Database</source>
        <translation>Nessun database</translation>
    </message>
    <message>
        <location filename="../qgiscloudplugindialog.py" line="477"/>
        <source>
To work with QGIS Cloud you need at least one QGIS Cloud database. Creating a database can take a few minutes, please be patient.

Do you want to create a new database now?
</source>
        <translation>
Per lavorare con QGIS Cloud è necessario almeno un database QGIS Cloud. La creazione di un database può richiedere alcuni minuti, sii paziente.

Vuoi creare un nuovo database ora?</translation>
    </message>
    <message>
        <location filename="../qgiscloudplugindialog.py" line="274"/>
        <source>The map name has been changed to &apos;{name}&apos; to conform to allowed characters ({allowed_chars}).</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../qgiscloudplugindialog.py" line="415"/>
        <source>Logged in as {0} ({1})
Paid until: {2}</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../qgiscloudplugindialog.py" line="445"/>
        <source>Invalid authentication token</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../qgiscloudplugindialog.py" line="957"/>
        <source>The following layer names do contain non allowed characters like &apos;, . : ;&apos; : {}. Please rename the layers and save the project before publishing.</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../qgiscloudplugindialog.py" line="965"/>
        <source>The following layer and/or group names are not unique in the project:

{}

Please rename the layers and/or the groups and save the project before publishing.</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../qgiscloudplugindialog.py" line="983"/>
        <source>The layer(s) {layerlist} have user defined CRS. The use of user defined CRS is not supported. Please correct the CRS before publishing!</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../qgiscloudplugindialog.py" line="995"/>
        <source>
The CRS of the background layer &apos;{layerName}&apos; is different to the  map CRS. This means that this layer will be reprojected in the published map and the webmap will therefore be slow. To improve this and make the webmap faster, go to

&apos;Project -&gt; Properties... -&gt; CRS&apos;

and set the map CRS to {layerCRS}. Continue publishing?</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../qgiscloudplugindialog.py" line="1026"/>
        <source>
The name of the QGIS project

%s

is invalid. It has the extension &apos;qgs.qgz&apos;. This is not allowed. Please correct the project name and publish the project again.</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../qgiscloudplugindialog.py" line="1190"/>
        <source>QGIS Cloud supports only Authority EPSG, IAU_2015 or ESRI reference systems</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../qgiscloudplugindialog.py" line="1335"/>
        <source>Table name too long</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../qgiscloudplugindialog.py" line="1335"/>
        <source>The name of table

{table_name}

has more than 62 characters and cannot be processed like this. Please give the table a shorter name.

Should the table name be shortened automatically?
</source>
        <translation type="unfinished"></translation>
    </message>
</context>
<context>
    <name>RasterUpload</name>
    <message>
        <location filename="../raster/raster_upload.py" line="100"/>
        <source>Database full</source>
        <translation>Database pieno</translation>
    </message>
    <message>
        <location filename="../raster/raster_upload.py" line="107"/>
        <source>Creating table &apos;{table}&apos;...</source>
        <translation>Creando tabella &apos;{table}&apos;...</translation>
    </message>
    <message>
        <location filename="../raster/raster_upload.py" line="100"/>
        <source>Upload would exceeded the maximum database size for your current QGIS Cloud plan. Please free up some space or upgrade your QGIS Cloud plan.</source>
        <translation>Il caricamento supererebbe la dimensione massima del database per il piano QGIS Cloud corrente. Libera un po&apos; di spazio o aggiorna il tuo piano QGIS Cloud.</translation>
    </message>
    <message>
        <location filename="../raster/raster_upload.py" line="131"/>
        <source>Creating overview-level {level} for table &apos;{table}&apos;...</source>
        <translation>Creazione di un livello di panoramica {level} per la tabella &apos;{table}&apos;....</translation>
    </message>
    <message>
        <location filename="../raster/raster_upload.py" line="145"/>
        <source>Registering raster columns of table &apos;%s&apos;...</source>
        <translation>Registrazione delle colonne raster della tabella &amp;quot;%s&amp;quot;...</translation>
    </message>
    <message>
        <location filename="../raster/raster_upload.py" line="646"/>
        <source>Uploading tiles...</source>
        <translation>Caricamento piastrelle...</translation>
    </message>
    <message>
        <location filename="../raster/raster_upload.py" line="708"/>
        <source>Calculating raster params for {sum_tiles} tiles ...</source>
        <translation>Calcolo dei parametri raster per le piastrelle {sum_tiles} ...</translation>
    </message>
    <message>
        <location filename="../raster/raster_upload.py" line="141"/>
        <source>Creating GIST Index for table &apos;{table}&apos;...</source>
        <translation type="unfinished"></translation>
    </message>
</context>
<context>
    <name>RelationSizeDialog</name>
    <message>
        <location filename="../ui_relation_size.ui" line="14"/>
        <source>Table sizes</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../ui_relation_size.ui" line="58"/>
        <source>Export CSV</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../ui_relation_size.py" line="79"/>
        <source>Total amount of used tablespace %s</source>
        <translation type="unfinished"></translation>
    </message>
</context>
<context>
    <name>map_settings</name>
    <message>
        <location filename="../mapsettings.ui" line="14"/>
        <source>Map Settings</source>
        <translation>Impostazioni della mappa</translation>
    </message>
    <message>
        <location filename="../mapsettings.ui" line="20"/>
        <source>General Settings</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../mapsettings.ui" line="53"/>
        <source>Map public</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../mapsettings.ui" line="104"/>
        <source>Viewer</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../mapsettings.ui" line="123"/>
        <source>Scales</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../mapsettings.ui" line="139"/>
        <source>Language</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../mapsettings.ui" line="152"/>
        <source>Viewer active</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../mapsettings.ui" line="165"/>
        <source>WMS public</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../mapsettings.ui" line="183"/>
        <source>Allowed Users</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../mapsettings.ui" line="235"/>
        <source>Search Settings</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../mapsettings.ui" line="246"/>
        <source>SQL for search with return values displaytext and bbox in map coordinates, e.g. </source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../mapsettings.ui" line="267"/>
        <source>Search type</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../mapsettings.ui" line="280"/>
        <source>Search DB</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../mapsettings.ui" line="313"/>
        <source>Search SQL</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../mapsettings.ui" line="331"/>
        <source>SELECT name AS displaytext, Box2D(st_transform(wkb_geometry, &lt;CRS&gt;)) AS bbox 
FROM searchtable 
WHERE name ILIKE &apos;%?%&apos;;</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../mapsettings.ui" line="343"/>
        <source>Share Map</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../mapsettings.ui" line="357"/>
        <source>Share with</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../mapsettings.ui" line="376"/>
        <source>CheckBox</source>
        <translation type="unfinished"></translation>
    </message>
</context>
</TS>
